bigtable/type.go

```diff
@@ -112,6 +122,14 @@ func (sum SumAggregator) fillProto(proto *btapb.Type_Aggregate) {
 	proto.Aggregator = &btapb.Type_Aggregate_Sum_{Sum: &btapb.Type_Aggregate_Sum{}}
 }
 
+type unknownAggregator struct {
+	wrapped *btapb.Type_Aggregate
+}
+
+func (ua unknownAggregator) fillProto(proto *btapb.Type_Aggregate) {
+	proto.Aggregator = ua.wrapped.Aggregator
```

Comment:
pointer nil check

Reply:
having basically no experience writing go, what am I supposed to do if its nil?